contributing.md

```diff
@@ -5,7 +5,7 @@ By participating in this project you agree to abide by its terms.
 
 ## Add quality
 
-Ensure your pull request adheres to the following guidelines
+Ensure your pull request adheres to the following guidelines :
```

Comment:
No space before the colon please.

I wonder why it seems no caution was taken when creating the PR and the update.